src/evrythng/entities/collections_.py

```diff
@@ -19,6 +19,23 @@
 }
 
 
+def create_collection_action(type_, collection=None, timestamp=None, identifiers=None,
+                             location=None, locationSource=None, context=None, customFields=None,
+                             api_key=None):
+    """Create an Action for a Collection"""
+    kwargs = locals()
+    kwargs['type'] = kwargs['type_']
+    del kwargs['type_']
+    api_key = kwargs.pop('api_key', None)
+
+    from evrythng.entities.actions import field_specs as action_field_specs
```

Comment:
Is this ok?  Seemed preferable to managing additional field_specs for Actions and Thngs.